Client/Frontend/Browser/BrowserViewController/BrowserViewController+URLBarDelegate.swift

```diff
@@ -369,8 +367,8 @@ extension BrowserViewController: URLBarDelegate {
 
             showFirefoxHome(inline: false)
         }
-
-        LeanPlumClient.shared.track(event: .interactWithURLBar)
+        
```

Comment:
In Xcode -> Preferences -> Text Editing -> Editing, make sure both `Automatically trim trailing whitespace` and `Including whitespace only lines` is enabled. Let's not have random empty tab chars everywhere. :)
```suggestion

```